apps/client-web/pages/clients/[id]/url.vue

```
@@ -1,7 +1,7 @@
 <script lang="ts">
 import { AClientScopes } from '@authup/client-web-kit';
 import type { Client, ClientScope } from '@authup/core-kit';
-import { VCFormInput, VCFormInputCheckbox } from '@vuecs/form-controls';
+import { VCFormInput, VCFormCheckbox } from '@vuecs/forms';
```

ESLint

[error] Member 'VCFormCheckbox' of the import declaration should be sorted alphabetically.

(sort-imports)